File: java/src/com/flyzipline/main/Flight.java
package com.flyzipline.main;

import java.util.List;

/**
 * Feel free to extend as needed
 */
public class Flight {
    private final int launchTime;
    private final List<Order> orders;

    public Flight(int launchTIme, List<Order> orders) {
        this.launchTime = launchTIme;
        this.orders = orders;
    }

    @Override
    public String toString() {
        return "Flight{" +
                "launchTime=" + launchTime +
                ", orders=" + orders +
                '}';
    }
}

File: ruby/lib/flight.rb
##
# Feel free to extend as needed
class Flight
  def initialize(launch_time, orders)
    @launch_time = launch_time
    @orders = orders
  end

  def to_s
    hospitals = @orders.map { |order| order.hospital.name }
    hospitals.join("->")
  end
end
